src/js/constants.js

```diff
@@ -23,13 +23,19 @@ export const MediaUIEvents = {
 };
 
 export const MediaUIAttributes = {
+  MEDIA_AIRPLAY_UNAVAILABLE: 'media-airplay-unavailable',
+  MEDIA_PIP_UNAVAILABLE: 'media-pip-unavailable',
   MEDIA_PAUSED: 'media-paused',
   MEDIA_MUTED: 'media-muted',
   MEDIA_VOLUME_LEVEL: 'media-volume-level',
   MEDIA_VOLUME: 'media-volume',
+  MEDIA_VOLUME_UNAVAILABLE: 'media-volume-unavailable',
```

Comment:
I'll say I'm torn between `media-volume-unavailable` and `media-volume-control-unavailable`. The first is nice and simple but not totally accurate, while the second is accurate but more verbose. Interested in other thoughts on that. That said, I lean towards keeping it as the simple version, because as far as the UI is concerned, we can't rely on any of the volume info, even to display the current volume.

Reply:
I favor `media-volume-unavailable` as well, nice and simple